pkg/terraform/executor.go

```diff
@@ -516,31 +516,31 @@ func (ex *Executor) logPath(id int) string {
 func (ex *Executor) checkVersion() error {
 	vOutput, err := ex.executeSync("--version")
 	if err != nil {
-		return fmt.Errorf("Error checking Terraform version: %w", err)
+		return fmt.Errorf("error checking terraform version: %w", err)
```

Comment:
I think `terraform` should be `Terraform` only. What do you think @invidian ?

Reply:
@knrt10 changes made